general/q5.py

```diff
@@ -0,0 +1,17 @@
+def areAnagrams(string1, string2):
```

Comment:
one thing your function names don't follow PEP8 style guide it should be are_anagrams

Reply:
Good use of comments though. Should look into docstrings and type hints for more best practice